src/utils/paymentQRHelper.test.ts

```diff
@@ -69,8 +69,8 @@ describe("tests for updateTransactionsPaymentQRIdentifiers", () => {
 
   it("should update identifiers properly", () => {
     expect.assertions(1);
-    updateTransactionsPaymentQRIdentifiers(transactions);
-    expect(transactions).toStrictEqual([
+    updateTransactionsPaymentQRIdentifiers(transactionsBeforeUpdate);
+    expect(transactionsBeforeUpdate).toStrictEqual([
```

Comment:
oooh, can be done later, should we modify the transactions in place, or we return a new array haha

Reply:
dont have to be in this PR, what are your thoughts?

Reply:
Changed it to return a new array in [c23dfa0](https://github.com/rationally-app/mobile-application/pull/530/commits/c23dfa03ac91de6a4f62da67b01f173225edd49a)!

I think it makes more sense to return a new array, since it is more obvious that the array was updated.